plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go

```diff
@@ -9,47 +9,66 @@ import (
 
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	conventions "go.opentelemetry.io/collector/semconv/v1.18.0"
+	semconv "go.opentelemetry.io/collector/semconv/v1.18.0"
```

Comment:
We're importing the same dependency under two names. `semconv` is the standard. Remove the other one.

Reply:
Updated.